### PR TITLE
Fix/36736 fix the color of upload status

### PR DESCRIPTION
**Pull request by sunnyssr**
This a fix for issue #36736
#### Changes proposed in this Pull Request

* Add color to upload status for importer

#### Testing instructions

*

Fixes #

### CODE REVIEW

**Comment by sunnyssr** [comment]
Hey @sixhours 
Can you take a look at it?
This is in reference to issue #36736

**Comment by sixhours** [comment]
Thanks for this PR! I'm not sure we're going to solve this just for Importers; we may want to look at the uploader component itself, to see if we can make improvements higher up in the chain. 

In the meantime, can you please add:

* Before and After screenshots of the changes you're proposing
* Instructions for testing your PR

It also looks like a commit from your other issue got mixed up with this one; could you rebase so this PR only includes the change related to this issue?

Thanks!

**Comment by sunnyssr** [comment]
@sixhours
I've removed commit from my other issue. Sorry for that. I didn't notice that while creating the pull request. 
Here are the before and after screenshot of the change I am proposing:
Before:-
screencapture-wordpress-import-sunnyssr-wordpress-com-2019-10-15-15_19_59
After:-
screencapture-calypso-localhost-3000-import-sunnyssr-wordpress-com-2019-10-15-15_29_40

File: client/my-sites/importer/uploading-pane.jsx [hunk 1]
```diff
@@ -145,7 +145,9 @@ class UploadingPane extends React.PureComponent {
 					onClick={ isReadyForImport ? this.openFileSelector : null }
 					onKeyPress={ isReadyForImport ? this.handleKeyPress : null }
 				>
-					<div className="importer__upload-content">
+					<div
+						className={ `importer__upload-content ${ this.props.importerStatus.importerState }` }
```
Comment (sixhours) [inline client/my-sites/importer/uploading-pane.jsx]: When combining classnames from a prop like this, you can use the `classnames` method. Rough example:

```
/**
 * External dependencies
 */
import classnames from 'classnames';

const classNames = classnames( 'importer__upload-content', this.props.importerStatus.importerState );

<div className={ classNames } />
```

A live example:
https://github.com/Automattic/wp-calypso/blob/ddb2a6a536d200635affce452094bd38fe571a02/client/reader/following-manage/feed-search-results.jsx#L58-L60

**Comment by sunnyssr** [comment]
@sixhours
I've made the changes. Please take a look.

**Comment by sunnyssr** [comment]
@sixhours 
I've fixed it. Here is a screenshot:
screencapture-calypso-localhost-3000-import-sunnyssr-wordpress-com-2019-10-15-22_37_20 (1)

**Pull request merged**